src/main/webapp/i18n/de/global.json

```diff
@@ -308,7 +308,12 @@
         "invalid.auxiliary.repository.description": "Die Beschreibung von mindestens einem Hilfs-Repository ist zu lang!",
         "invalid.testcases.weights": "Vor Deaktivierung der manuellen Bewertung muss für mindestens einen Testfall die Bewertungsgewichtung größer Null gesetzt werden.",
         "idnull": "Ungültige ID",
-        "unexpectedError": "Ein unerwarteter Fehler ist aufgetreten: {{error}}"
+        "unexpectedError": "Ein unerwarteter Fehler ist aufgetreten: {{error}}",
+        "manualCodeHintOperation": "Dieser Endpoint erlaubt keine manuelle Bearbeitung von Codehinweisen.",
+        "exerciseNotDefined": "Die Aufgabe ist nicht definiert.",
+        "exerciseIdMismatch": "Die Aufgaben-IDs stimmen nicht überein.",
+        "solutionEntryIdMismatch": "Die Solution-Entries-IDs stimmen nicht überein.",
+        "solutionEntryTestCaseMismatch": "Der Tests gehört nicht zu dem SolutionEntry."
```

Comment:
You expose here implementation details that the user doesn't know about. Please translate it so any user understands the issue.

Reply:
And why are these in the global file?

Reply:
I changed the description to be more general. The files are in the global.json file because all the translations (name, description, error) for every button on the exercise detail page are stored in this file. I thought sticking with this may be the best solution.